components/TaskCard.vue

```diff
@@ -83,14 +90,13 @@
 <script>
 import { mapGetters } from 'vuex';
 import { SUCCESS, ERROR } from '../constants';
-import DeleteIcon from '@/assets/svg/Delete.svg';
-import LoadingIcon from '@/components/buttons/LoadingIcon.vue';
-import EditIcon from '@/assets/svg/Edit.svg';
-import TickIcon from '@/assets/svg/Tick.svg';
-import global from '@/mixins/global';
-
+import DeleteIcon from '../assets/svg/Delete.svg';
+import EditIcon from '../assets/svg/Edit.svg';
+import TickIcon from '../assets/svg/Tick.svg';
+import global from '../mixins/global';
+import LoadingIcon from './buttons/LoadingIcon.vue';
```

Comment:
Can we use relative path here?

Reply:
Adding relative path throughs error on test.

Reply:
resolved

Reply:
resolved